### DISCUSSION TITLE
EventToCommandBehavior Label Tapped Event

### DISCUSSION THREAD

**Discussion by Galatis-Giuliano** [xamarin/XamarinCommunityToolkit#1313] [Q&A]
I have been trying to use the EventToCommandBehavior functionality on the tapped event of a label but I'm not sure how to proceed. Label.Behaviors does not contain the tapped event instead it is in Label.GestureRecognizers. Would EventToCommandBehavior still work in this case?

**Comment by acaliaro** [comment]
I think it does not work because Label has not a "tapped" event. You can use GestureRecognizer to handle it, or TouchEffect from XCT
https://github.com/xamarin/XamarinCommunityToolkit/pull/566
